### PR TITLE
Add CVE-2017-5645

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by vasulakkaraju** [comment]
updated CVE info per the Ref : https://issues.apache.org/jira/browse/LOG4J2-1863

File: database/java/2017/5645.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - https://access.redhat.com/errata/RHSA-2017:3244
+    - https://access.redhat.com/errata/RHSA-2017:3399
+    - https://access.redhat.com/errata/RHSA-2017:3400
+    - https://issues.apache.org/jira/browse/LOG4J2-1863
+
+affected:
+    - groupId: org.apache.logging.log4j
+      artifactId: log4j-api
+      version:
+        - "<= 2.0 , 2.8.1"
```
Comment (msrb) [inline database/java/2017/5645.yaml]: Whitespaces shouldn't be here and the versions should be the other way around:

```
- "<= 2.8.1,2.0"
```

File: database/java/2017/5645.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      fixedin:
+        - "== 2.8.2"
```
Comment (msrb) [inline database/java/2017/5645.yaml]: `>=`

**Comment by vasulakkaraju** [comment]
@msrb PTAL

**Comment by msrb** [comment]
LGTM, thanks :wink:

**Pull request merged**